azure-pipelines.yml

```diff
@@ -174,7 +176,7 @@ stages:
           msbuildArgs: '/p:Sha1=$(Build.SourceVersion) /p:BranchName=$(Build.SourceBranchName) /p:BuildNumber=$(Build.BuildId) /p:BuildConfiguration=$(BuildConfiguration)'
 
       - task: SonarCloudPrepare@1
-        displayName: 'Run Code Analysis - Begin Step (Pull Request)'
+        displayName: 'Code Analysis - Begin Step (Pull Request)'
```

Comment:
I think we can remove also the "Step" and use PR instead of Pull Request.

Reply:
I did the same for "End Step" as well to be consistent.